templates/base.html

```diff
@@ -0,0 +1,22 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1">
```

Comment:
There is a missing opening `<html>` tag at the beginning of the document. Make sure to include it to ensure the HTML document is well-formed.